src/Updater.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Tuf\ComposerIntegration;
+
+use Psr\Http\Message\StreamInterface;
+use Tuf\Client\Updater as BaseUpdater;
+use Tuf\Metadata\TargetsMetadata;
+
+/**
+ * Defines an updater that exposes additional information about TUF targets.
+ */
```

Comment:
I think we should have comment here about why we are extending this class and maybe a class name that reflects.

It not only expose extra information but exposed the ability to verify targets with the updater downloading them itself, correct?

I also don't know if we have an `@internal` policy but we should probably create one. Maybe everything should be `@internal` but I think this should definitely be. This is not meant as a general class to get around the `php-tuf` being responsible for downloads but something we *have* to do because of the way Composer works.

Reply:
Agreed. Made it internal and documented why.